src/components/responsive-table/cell.tsx

```diff
@@ -51,6 +55,9 @@ const Cell = ({
       className={cx('tableCell', className)}
       style={{
         maxWidth: `${minWidth}px`,
+        width: `${
+          accessor === visibleColumns[visibleColumns.length - 1]?.accessor ? 0 : (minWidth / containerWidth) * 100
```

Comment:
`visibleColumns[visibleColumns.length - 1]` === last column?

Reply:
ye, `visibleColumns[visibleColumns.length - 1]` its a last column, so, the logic looks like this: i check if our cell not in the last column => i give it `width` that === `minWidth` percentages of `containerWidth`

Reply:
the point why we need `containerWidth`: my first idea was to give all columns same width besides last one so i did calculation like `100` (100% of our table) / `amountOfColumn - 1` (all columns without last one), but then i understand that same width for all columns makes no sense, because some column may contain only 1 symbol and other 1 may containt 15 and we will cut the longed one but the shorter one will have a lot of white space so then i decided to count how much percentages our column should be using our `minWidth` prop and full `containerWidth` prop